packages/shared/src/components/layout/HeaderLogo.tsx

```diff
@@ -19,14 +21,15 @@ function HeaderLogo({
 }: HeaderLogoProps): ReactElement {
   const featureTheme = useFeatureTheme();
   const { user } = useAuthContext();
+  const showPlusSubscription = useFeature(feature.plusSubscription);
```

Comment:
again useConditionalFeature enrol if user isPlus is on, it will enrol any time user sees the header this way

Reply:
Removed it from here